project/plugins.sbt

```diff
@@ -4,6 +4,7 @@ addSbtPlugin("com.github.gseitz" % "sbt-protobuf" % "0.6.3")
 addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.7")
 addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.1.0")
 addSbtPlugin("org.scalastyle" % "scalastyle-sbt-plugin" % "1.0.0")
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.0")
```

Comment:
If you have some time would be nice to bump some of the other plugin versions as well but not necessary. I think scoverage and scalastye have new versions